service/templates/admin.html

```diff
@@ -822,6 +823,25 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									</md-input-container>
 								</md-card-content>
 							</md-card>
+            </section>
+
+            <section layout="row" flex>
+							<md-card flex=50 id="google_analytics" style="height:280px">
+								<md-card-title>
+									<md-card-title-text>
+										<span>Google Analytics</span>
+									</md-card-title-text>
+									<md-button ng-click="clearSection('splunk')">
```

Comment:
does this button do what we want it to do?

Reply:
fixed